landscape/lib/network.py

```diff
@@ -138,20 +139,30 @@ def get_flags(sock, interface):
     return struct.unpack("H", data[16:18])[0]
 
 
-def get_active_device_info(skipped_interfaces=("lo",),
-                           skip_vlan=True, skip_alias=True, extended=False):
+def get_virtual():
+    """
+    Returns a list of virtual interfaces see link:
+    https://unix.stackexchange.com/a/58328
+    """
+    try:
+        interfaces = os.listdir('/sys/devices/virtual/net')
+    except OSError:
+        interfaces = []
+    return interfaces
+
+
+def get_active_device_info(skip_alias=True, extended=False):
     """
     Returns a dictionary containing information on each active network
     interface present on a machine.
```

Comment:
This comment isn't accurate anymore, since you now list "active **physical** network interfaces".